### PR TITLE
Bakhat | restaurant | sprint-2

### PR DESCRIPTION
**Pull request by BakhatBegum**
## Learners, PR Template

Self checklist

- [ ] I have committed my files one by one, on purpose, and for a reason
- [ ] I have titled my PR with COHORT_NAME | FIRST_NAME LAST_NAME | REPO_NAME | WEEK 
- [ ] I have tested my changes
- [ ] My changes follow the [style guide](https://syllabus.codeyourfuture.io/guides/code-style-guide/)
- [ ] My changes meet the [requirements](./README.md) of this task

## Changelist

The task was to fix the test case.

## Questions

Ask any questions you have for your reviewer.

### CODE REVIEW

File: src/components/App/App.jsx [hunk 1]
```diff
@@ -14,9 +14,9 @@ const App = () => {
 
   return (
     <div className="app">
-      <AppHeader/>
+      <AppHeader />
       <Bookings />
```
Comment (PERicci) [inline src/components/App/App.jsx:18]: I believe the `<Restaurant />` component was supposed to be placed here, below Bookings, according to the instructions provided in the issue.

File: src/components/App/App.test.jsx [hunk 2]
```diff
@@ -5,7 +5,7 @@ import { render, screen } from "@testing-library/react";
 describe("App", () => {
   it("renders the CYF Hotel header", () => {
     render(<App />);
-    const h1 = screen.getByRole("heading", { level: 1 });
-    expect(h1).toBeInTheDocument();
+    const headerElement = screen.getByRole("heading", { name: "CYF Hotel" });
+    expect(headerElement).toBeInTheDocument();
   });
```
Comment (PERicci) [inline src/components/App/App.test.jsx:10]: I found this approach interesting. The only thought that crossed my mind is: If we decide to change the text in `<h1>`, we will have to change the test too.
So, in this situation, I do not know which is better: Being more general using `<h1>` or being more specific using the proper title.

File: src/components/Restaurant/Restaurant.test.jsx [hunk 3]
```diff
@@ -6,7 +6,7 @@ describe("Restaurant", () => {
   it("renders an Orders heading", () => {
     render(<Restaurant />);
     const titleElement = screen.getByRole("heading", {
-      level: 3,
+      name: 'Restaurant Orders',
```
Comment (PERicci) [inline src/components/Restaurant/Restaurant.test.jsx:9]: This is the same situation I commented on in the code above. I don't know which approach is better between the heading's content and the heading's level.

**Review by PERicci — CHANGES_REQUESTED** [review]
`<Restaurant />` component is missing in App.jsx.
The comments I made are about questions that we can discuss during the week or in class on Saturday since they do not interfere with the page's functionality.

File: src/components/App/App.jsx (hunk 1 above)
Comment (BakhatBegum) [reply src/components/App/App.jsx:18]: Thank you very much @PERicci, I made changes in App.jsx as per your feedback.

**Pull request closed without merging**